app/models/user.rb

```diff
@@ -8,4 +9,11 @@ class User < ApplicationRecord
   has_many :answers
   has_many :weekly_activities
   enum :base_office, {braga: 0, coimbra: 1}
+  after_commit :add_default_picture, on: [:create]
+
+  private def add_default_picture
```

Comment:
```suggestion
  private 
  
  def add_default_picture
```

Usually we write the private and then all methods that are private below it, since it's not a per method thing